cssom-view/htmlelement-offset-width-001.html

```diff
@@ -19,8 +19,11 @@
             el.appendChild(document.createTextNode("..."));
             assert_equals(el.offsetWidth, 0, "new element has offset width not equal to zero");
             assert_not_equals(document.getElementById("myDiv"), 0, "element with content has offset width equal to zero");
-            document.documentElement.remove();
+            var html = document.removeChild(document.documentElement);
             assert_equals(el.offsetWidth, 0, "new element has offset width not equal to zero after removing documentElement");
+
+            // Put back the root element, so that testharnessreport.js can append the result.
+            document.appendChild(html);
```

Comment:
This should be in `add_cleanup()` before the assert because otherwise if the assert fails this isn't reached.